src/Faker/Provider/Base.php

```diff
@@ -15,9 +15,9 @@ class Base
     protected $generator;
 
     /**
-     * @var \Faker\UniqueGenerator
+     * @var \Faker\UniqueGenerator|null
      */
-    protected $unique;
+    protected $unique = null;
```

Comment:
This change is not needed. Uninitialized properties are already set to null.

Reply:
True, habit from property types.

```suggestion
    protected $unique;
```